NEWS.md

```diff
@@ -1,4 +1,17 @@
-## 1.5.0 - Unreleased
+## 2.1.0 - Unreleased
+
+## 2.0.0 - Released (Orchid R1 2023)
+The focus of this release was to update to Java 17 and improvement Logging
+
+[Full Changelog](https://github.com/folio-org/mod-orders/compare/v1.4.0...v2.0.0)
+
+### Stories
+* [MODEBSNET-51](https://issues.folio.org/browse/MODEBSNET-51) Update to Java 17
```

Comment:
[MODEBSNET-52](https://issues.folio.org/browse/MODEBSNET-52) Update the module to Spring boot v3.0.0 and identify issues.

This also